README.md

```diff
@@ -1,3 +1,37 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+Welcome to my Northcoders Backend Project!
+
+To follow along with me, you will need to do a few things before getting started.
+
+Go to my online repository and create your own fork:
+https://github.com/prgf87/nc-news-project
+
+Then, navigate to your local directory and clone my repo by using the command below.
+
+`git clone https://github.com/prgf87/nc-news-project`
+
+Navigate into your newly cloned repo and use `code .` to launch VSCode.
+
+Run `npm i` in your terminal to install all the necessary dependecies (make sure to navigate into your newly cloned folder before running this command!)
+
+You will also have to create two .env files(.env.test & .env.development), one for test purposes, and one for the development/deployment stage.
```

Comment:
I wouldn't mention 'deployment' yet, as for that purpose we're gonna have a .env.production file which we don't include here at this point.